S2VX.Game/Story/Note/Notes.cs

```diff
@@ -16,6 +16,7 @@ public class Notes : CompositeDrawable {
         public float FadeInTime { get; set; } = 100;
         public float ShowTime { get; set; } = 1000;
         public float FadeOutTime { get; set; } = 100;
+        public int MissThreshold { get; set; } = 200;
```

Comment:
I would only put Command modify-able properties inside of Notes. MissThreshold might be better as a const inside of GameNote for now.